test/FrameTest.cpp

```diff
@@ -205,6 +205,19 @@ TEST_F(FrameTest, Frame_LEASE) {
   EXPECT_EQ(numberOfRequests, frame.numberOfRequests_);
 }
 
+TEST_F(FrameTest, Frame_REQUEST_RESPONSE) {
+  uint32_t streamId = 42;
+  FrameFlags flags = FrameFlags_METADATA;
```

Comment:
maybe more interesting that if we don't set this flag, that it gets correctly set.